AdminServer/appscale/admin/__init__.py

```diff
@@ -937,6 +954,12 @@ def update_version(self, project_id, service_id, version_id, new_fields):
     version = json.loads(version_json)
     new_ports = utils.assign_ports(version, new_fields, self.zk_client)
     version['appscaleExtensions'].update(new_ports)
+    if not 'manualScaling' in version:
```

Comment:
If a user has manual scaling defined for their version and they decide to change to automatic scaling, it looks like the patch call would succeed without any side effect.